src/test/fuzz/fuzz.cpp

```diff
@@ -72,7 +72,10 @@ extern "C" int LLVMFuzzerInitialize(int* argc, char*** argv)
 }
 
 #if defined(PROVIDE_MAIN_FUNCTION)
-__attribute__((weak)) int main(int argc, char** argv)
+#ifndef WIN32
+__attribute__((weak))
```

Comment:
Hmm, I think we don't even need the attribute at all, now that we have a proper configure check.

* Afl does need the main function (preferably without the attribute)
* honggfuzz and libFuzzer shouldn't get it
* raw builds need the main function (preferably without the attribute)

Reply:
That sounds right. Perhaps that will save me from having to figure out where this mac failure is coming from.